src/CommandLine.hs

```diff
@@ -157,5 +164,9 @@ pFees =
         <> metavar "NATURAL"
     )
 
+pMaybe :: Parser a -> Parser (Maybe a)
+pMaybe parser =
```

Comment:
Is this just [`optional`](https://hackage.haskell.org/package/base-4.16.0.0/docs/Control-Applicative.html#v:optional)?